### PR TITLE
feat : load firmware from github (cors needed)

### PR DESCRIPTION
**Pull request by jsy-rg1570**

### CODE REVIEW

**Comment by jsy-rg1570** [comment]
image

File: index.js [hunk 1]
```diff
@@ -198,8 +206,20 @@ connectButton.onclick = async () => {
...
         try {
+          var response;
+          if(!diymodelsel.value.includes("https://github.com/BitMaker-hub/NerdMiner_v2")){
             addToLog(`Fetching: assets/${diymodelsel.value}/${item.fileName}`);
-            const response = await fetch(`assets/${diymodelsel.value}/${item.fileName}`);
+            response = await fetch(`assets/${diymodelsel.value}/${item.fileName}`);
+          } else {
+            if(item.fileName.includes("boot_app0.bin")){              
+              addToLog(`Fetching: ${diymodelsel.value.substring(0, diymodelsel.value.lastIndexOf("/") + 1)}${item.fileName}`);
+              response = await fetch(`https://cors-anywhere.herokuapp.com/${diymodelsel.value.substring(0, diymodelsel.value.lastIndexOf("/") + 1)}${item.fileName}`);
```
Comment (jsy-rg1570) [inline index.js:216]: @BitMaker-hub workaround about CORS

File: index.html [hunk 2]
```diff
@@ -124,6 +125,35 @@ <h3 id="success"></h3>
...
                 ];
 
+                function addGithubReleases() {
+                    
+                    fetch("https://api.github.com/repos/BitMaker-hub/NerdMiner_v2/tags")
+                    .then(response => response.json())                        
+                    .then(tags => tags
+                        .map(tag => tag.name)
+                        .filter(tag => ["nerdminer-prerelease", "nerdminer-release"].some(val => tag.includes(val)))
+                        .slice(0, 2)
```
Comment (jsy-rg1570) [inline index.html:135]: @BitMaker-hub to load 3 tags

**Review by BitMaker-hub — COMMENTED** [review]
perfect

**Pull request merged**